### PR TITLE
Clarify how auto imports work in Python

### PR DESCRIPTION
**Pull request by luabud**
For https://github.com/microsoft/pylance-release/issues/5996

Adding a screenshot and clarifying a bit the wording on our docs to make it clearer how auto import suggestions work

### CODE REVIEW

File: docs/python/editing.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ Enabling the full set of IntelliSense features by default could end up making yo
 
 ### Enable Auto Imports
 
-Pylance offers auto import suggestions for modules in your workspace and/or packages you have installed in your environment. This enables import statements to be automatically added as you type. Auto imports are disabled by default, but you can enable them by setting `python.analysis.autoImportCompletions` to `true` in your settings.
+Pylance offers auto import suggestions for modules in your workspace and/or packages you have installed in your environment. This enables import statements to be automatically added once you accept the suggestion that pops up as you type. Auto imports are disabled by default, but you can enable them by setting `python.analysis.autoImportCompletions` to `true` in your settings.
```
Comment (ntrogh) [inline docs/python/editing.md:39]:
```suggestion
Pylance offers auto import suggestions for modules in your workspace and for packages you installed in your environment. As you type in the editor, you might get completion suggestions. When you accept the suggestion, auto import automatically adds the corresponding import statement to your file.

You can enable auto imports by setting `python.analysis.autoImportCompletions` to `true` in your settings. By default, auto imports are disabled.
```

Comment (ntrogh) [reply docs/python/editing.md:39]: Made the text a bit more wordy to avoid confusion.

File: docs/python/images/editing/auto-import-suggestion.png
Comment (ntrogh) [inline docs/python/images/editing/auto-import-suggestion.png:1]: This image file is not loading for me.

**Review by ntrogh — APPROVED** [review]
@luabud I expanded the text a bit, but feel free to go with what you had. I also have problems viewing the screenshot...

**Review by cwebster-99 — DISMISSED** [review]

File: docs/python/images/editing/auto-import-suggestion.png
Comment (luabud) [reply docs/python/images/editing/auto-import-suggestion.png:1]: thank you for the suggestion and bringing this up! not sure what had happened with the image, but seems like it's fixed now!

**Review by FTOSevenK — APPROVED** [review]
#7384

**Review by ntrogh — APPROVED** [review]
LGTM! Thanks @luabud

**Pull request merged**